usl_pipeline/cloud_functions/requirements.txt

```diff
@@ -9,6 +9,7 @@ click-plugins==1.1.1
 cligj==0.7.2
 cloudevents==1.10.1
 deprecation==2.1.0
+fiona==1.9.6
```

Comment:
this will break the cloud function if it requires GDAL to be present during pip install time :'(

Reply:
okay! We talked and @rsutormin , you correctly pointed out that it would break the build if this was an issue!

We might still want to move the shape files readers out of `usl_lib` and into `map_uploader` because the cloud function would break if it ever attempted to call the shape file reader, as GDAL won't be present.

Reply:
I moved this code to map_uploader and removed dependency to fiona lib from usl_lib and from cloud_functions.